BuildUtils.cmake

```diff
@@ -24,17 +24,32 @@ if(PLATFORM_WIN32 OR PLATFORM_UNIVERSAL_WINDOWS)
                     "\"$<TARGET_FILE_DIR:${TARGET_NAME}>\"")
         endforeach(DLL)
 
-        # Copy D3Dcompiler_47.dll 
+        # Copy D3Dcompiler_47.dll and dxcompiler.dll
         if(MSVC)
-            if(WIN64)
-                set(D3D_COMPILER_PATH "\"$(VC_ExecutablePath_x64_x64)\\D3Dcompiler_47.dll\"")
+            if (${CMAKE_SIZEOF_VOID_P} EQUAL 8)
+                set(D3D_COMPILER_PATH "\"$(WindowsSdkDir)\\bin\\${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}\\x64\\D3Dcompiler_47.dll\"")
```

Comment:
Known issue:
`$(WindowsSdkDir)` and `CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION` are Visual Studio variables. Another solution needed to support all IDE's.